docs/docs.rst

```diff
@@ -0,0 +1,37 @@
+==========================
+Updating The Documentation
+==========================
+
+You need to have Python to build the documentation locally.
+
+The documentation is built in our continuous integration every time
```

Comment:
This sentence is weird, is it missing something? Maybe a final `.`? or every time a change is made`?

Reply:
Feel free to edit it directly. I rushed almost all the little I wrote.